test/conformance/kuttl/validate/clusterpolicy/cornercases/ephemeral-containers/resource-assert.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mypod
```

Comment:
I would give it a unique name if the pod cannot be cleaned up when something unexpected happens.